src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, ElementRef, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
```

Comment:
This looks identical to the original. Why is it needed?

Reply:
its been created in preperation for another ticket i will be working on this sprint. i was abit too eager!